### ISSUE TITLE
Being able to use Usingipad to drag components into canvas

### ISSUE THREAD

**Issue by HFolkertH** [saltcorn/saltcorn#2299]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Ingenu1ty** [comment]
Yep, a lot of people are [interested in this feature](https://github.com/saltcorn/saltcorn/issues/1954#issue-1847311433)
